app/models/reward.rb

```diff
@@ -4,5 +4,7 @@ class Reward < ApplicationRecord
   validates :price, numericality: { greater_than_or_equal_to: 1 }
   validates :title, :description, :price, presence: true
 
-  has_many :orders, dependent: :nullify
+  has_many :orders, dependent: :restrict_with_error
```

Comment:
I had to secure the rewards so that they could not be removed if ordered.

Reply:
is it really necessary here? in this case you can only delete an order if it has no categories - I don't think we should care about it while deleting an order + it is not in the Acceptance Criteria

Reply:
Returned to its previous status.